### ISSUE TITLE
React DOM UMD always warns in React 18

### ISSUE THREAD

**Issue by gaearon** [facebook/react#24221]
This is a false positive.

<img width="952" alt="Screenshot 2022-03-30 at 18 25 00" src="https://user-images.githubusercontent.com/810438/160895998-70db1fa4-e4f2-4289-9688-1822b6fb7dc6.png">

**Comment by RayJason** [comment]
Have you updated react-dom?  
- "react": "^18.0.0"
-  "react-dom": "^18.0.0"
 
```TypeScript
import { createRoot } from 'react-dom/client'
import App from './pages/app'

createRoot(document.getElementById('root')!).render(<App />)
```

**Comment by gaearon** [comment]
This is specifically about the UMD build.

**Comment by gaearon** [comment]
fixed by https://github.com/facebook/react/pull/24274

**Comment by gaearon** [comment]
Fix out in 18.1.0.
